kafka-connect-s3/src/main/java/io/confluent/connect/s3/S3SinkConnectorConfig.java

```diff
@@ -63,6 +63,11 @@
   public static final String REGION_CONFIG = "s3.region";
   public static final String REGION_DEFAULT = Regions.DEFAULT_REGION.getName();
 
+  public static final String S3_PROXY_HOST = "s3.proxy.host";
```

Comment:
But AWS API ClientConfiguration accepts hostname and port separately. If we accept as single string then it requires splitting.